src/index.js

```diff
@@ -26,22 +28,25 @@ class RetryChunkLoadPlugin {
         `;
 
           const getCacheBustString = () =>
-            this.options.cacheBust
-              ? `
+            this.options.cacheBust ?
+            `
             (${this.options.cacheBust})()
-          `
-              : '"cache-bust=true"';
+          ` :
+            '"cache-bust=true"';
 
           const isRetryWithCacheBustingQSPEnabled = () =>
-              this.options.isRetryWithCacheBustingQSPEnabled ?
-              `(${this.options.isRetryWithCacheBustingQSPEnabled})()` : false;
+            this.options.isRetryWithCacheBustingQSPEnabled ?
+            `(${this.options.isRetryWithCacheBustingQSPEnabled})()` : false;
+
+          const retryCallback = (errorType) => this.options.retryCallback ?
```

Comment:
Recommend renaming the retryCallback to preRetryCallback to make sure that the callback is not the one actually doing the retry.